content/tuf-root-update.md

```diff
@@ -0,0 +1,34 @@
++++
+title = "Sigstore Announcement: New TUF Trust Root and Client Compatibility"
+date = "2024-03-14"
+tags = ["sigstore","cosign","tuf"]
+draft = false
+author = "Sigstore TSC"
+type = "post"
++++
+
+## New TUF Trust Root
+
+We are planning to publish a new TUF trust root for Sigstore. This update does not contain any functional changes,
+but it does update to the latest version of the TUF specification.
+This means that older clients may not be able to load it properly. The current compatibility is as follows:
+
+* Cosign
+   - **v2.2.0+** (Released Aug 31st 2023) works, older Cosign v2 clients will not work
```

Comment:
Should you explicitly say “releases greater than or equal to v2.2.0?

Reply:
Updated